chroma-agent/chroma_agent/device_plugins/linux_components/block_devices.py

```diff
@@ -90,11 +95,20 @@ def as_device(x):
         'serial_83': x.get('IML_SCSI_83'),
         'size': int(get_default('IML_SIZE', 0, x)) * 512,
         'filesystem_type': x.get('ID_FS_TYPE'),
+        'filesystem_usage': x.get('ID_FS_USAGE'),
         'device_type': x.get('DEVTYPE'),
         'device_path': x.get('DEVPATH'),
         'partition_number': x.get('ID_PART_ENTRY_NUMBER'),
         'is_ro': x.get('IML_IS_RO'),
-        'parent': None
+        'parent': None,
+        'dm_multipath': x.get('DM_MULTIPATH_DEVICE_PATH'),
+        'dm_lv': x.get('DM_LV_NAME'),
+        'dm_vg': x.get('DM_VG_NAME'),
+        'dm_uuid': x.get('DM_UUID'),
```

Comment:
Is this code still relevant?

I removed dm_uuid in favor of vg_uuid and lv_uuid I think.

Reply:
addressed in later commits